mesheryctl/internal/cli/root/system/check.go

```diff
@@ -86,7 +85,7 @@ func runDockerHealthCheck(isPreRunExecution bool, subcommand string, platform st
 	//Check whether docker daemon is running or not
 	err := exec.Command("docker", "ps").Run()
 	if err != nil && isPreRunExecution { // if this is PreRunExec we trigger self installation
-		log.Warn("!! Docker is not running")
+		fmt.Printf("!! Docker is not running\n")
```

Comment:
Is this change necessary ? we can always read STDOUT and STDERR from the command and look for these errors or warnings.

Reply:
Thanks for raising this question. Two things, Logrus actually logs with additional details such as the time of logging, which makes it hard to match with the expected output stored in golden files.
Btw we will eventually need to find an alternate technique as we have logrus all over the project.

Reply:
Hey @hexxdump I think I found a solution for matching logrus outputs. Could you revisit the changes?